installer/install.iss

```diff
@@ -102,7 +102,7 @@ Filename: {app}\ReleaseNotes.html; Description: View Release Notes; Flags: shell
 [Files]
```

Comment:
Whoa. This comment was meant to be specific to the `innosetup: use new render-release-notes.sh script` commit, but I now see that GitHub anchored the comment to the PR's diff instead, so that it shows up in unrelated commits, too. Bummer.